src/server/cards/seaside/outpost.js

```diff
@@ -4,13 +4,32 @@ export default class Outpost extends Card {
   static cost = { coin: 5 };
   static types = new Set(['Action', 'Duration']);
   async onPlay(player) {
-    this.ignoreCleanUp = true;
-    player.playArea.filter(c => c !== this).forEach(c => {
-      if (c.title === 'Outpost') this.ignoreCleanUp = false;
-    });
-    if (player.game.previousPlayer === player) this.ignoreCleanUp = false;
+    if (
+      !player.cardsPlayedThisTurn.some(card => card !== this && card.title === 'Outpost') &&
+      !player.playArea.some(card => card !== this && card.title === 'Outpost')
```

Comment:
Still needs the check that the previous player wasn't the same player.

Reply:
is there a way to remove outpost from the play area on the 2nd turn? if not, isnt it good enough to make sure that there are no outposts in the play area?

Reply:
No there isn't and that would work. Except for Mission an Possession will mess with us. You can't outpost on a mission turn or after someone possessed you last turn.

Not sure if we ever make possession but should still plan for it's possibility.

Reply:
ok will fix later